encfs/FileUtils.cpp

```diff
@@ -946,7 +949,8 @@ static bool selectExternalChainedIV() {
         "If a file is renamed, it will not decode sucessfully unless it\n"
         "was renamed by encfs with the proper key.\n"
         "If this option is enabled, then hard links will not be supported\n"
-        "in the filesystem."));
```

Comment:
Instead of adding to translation strings, let's add it to the man page documentation instead.